src/Stratis.Features.FederatedPeg/FederatedPegFeature.cs

```diff
@@ -284,7 +283,6 @@ public static IFullNodeBuilder AddFederatedPeg(this IFullNodeBuilder fullNodeBui
                         {
                             services.AddSingleton<IRewardDistributionManager, RewardDistributionManager>();
                             services.AddSingleton<ICoinbaseSplitter, PremineCoinbaseSplitter>();
-                            services.AddSingleton<BlockDefinition, FederatedPegBlockDefinition>();
```

Comment:
This was a bug where when the `AddFederatedPeg` method add the `FederatedPegBlockDefinition` it gets overwritten with `AddPoACollateralMiningCapability` which adds the `SmartContractPoABlockDefinition`